pkg/bootstrap/bootstrapkubeconfig.go

```diff
@@ -281,15 +360,32 @@ func getKubeAPIServerCAData(ctx context.Context, clientHolder *helpers.ClientHol
 
 	// failed to get the ca from ocp, fallback to the kube-root-ca.crt configmap from the pod namespace.
 	klog.Info(fmt.Sprintf("No ca.crt was found, fallback to the %s/kube-root-ca.crt", caNamespace))
-	return getCABundleFromConfigmap(ctx, clientHolder, caNamespace)
+	return getKubeRootCABundle(ctx, clientHolder, caNamespace)
+}
+
+func getKubeRootCABundle(ctx context.Context, clientHolder *helpers.ClientHolder,
+	caNamespace string) ([]byte, error) {
+	return getCABundleFromConfigmap(ctx, clientHolder, caNamespace, "kube-root-ca.crt", "ca.crt")
 }
 
-func getCABundleFromConfigmap(ctx context.Context, clientHolder *helpers.ClientHolder, caNamespace string) ([]byte, error) {
-	rootCA, err := clientHolder.KubeClient.CoreV1().ConfigMaps(caNamespace).Get(ctx, "kube-root-ca.crt", metav1.GetOptions{})
+func getCABundleFromConfigmap(ctx context.Context, clientHolder *helpers.ClientHolder,
+	caNamespace, caName string, caKeys ...string) ([]byte, error) {
+	rootCA, err := clientHolder.KubeClient.CoreV1().ConfigMaps(caNamespace).Get(ctx, caName, metav1.GetOptions{})
 	if err != nil {
 		return nil, err
 	}
-	return []byte(rootCA.Data["ca.crt"]), nil
+
+	if len(caKeys) == 0 {
+		caKeys = []string{"ca-bundle.crt", "ca.crt", "tls.crt"}
+	}
+
+	for _, key := range caKeys {
+		if data, ok := rootCA.Data[key]; ok {
+			return []byte(data), nil
```

Comment:
If multiple keys exit, the values should be merged.

Reply:
ok, but why users put multiple bundles into different keys？ in that case, should they use multiple configmaps?

Reply:
Yes, you are right. It's might not be a good idea to merge the value of different keys.